bridge/client/app/_components/ktb-sequence-state-list/ktb-sequence-state-list.component.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { ComponentFixture, fakeAsync, TestBed, waitForAsync } from '@angular/core/testing';
+
+import { KtbSequenceStateListComponent } from './ktb-sequence-state-list.component';
+import {HttpClientTestingModule} from "@angular/common/http/testing";
+import {AppModule} from "../../app.module";
+import {KtbSequenceTimelineComponent} from "../ktb-sequence-timeline/ktb-sequence-timeline.component";
+
+describe('KtbServicesListComponent', () => {
+  let component: KtbSequenceStateListComponent;
+  let fixture: ComponentFixture<KtbSequenceStateListComponent>;
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [],
+      imports: [
+        AppModule,
+        HttpClientTestingModule,
+      ],
+    })
+      .compileComponents()
+      .then(() => {
+        fixture = TestBed.createComponent(KtbSequenceStateListComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+      });
+  }));
+
+  afterEach(fakeAsync(() => {
+    fixture.destroy();
+    TestBed.resetTestingModule();
+  }));
+});
```

Comment:
Please add a test for the dataSource (updateDataSource) as it would make sense for this component ;)